mods/cnc/rules/defaults.yaml

```diff
@@ -627,6 +633,8 @@
 ^Plane:
 	Inherits@1: ^ExistsInWorld
 	Inherits@2: ^SpriteActor
+	UpdatesPlayerStatistics:
+		AddToArmyValue: true
```

Comment:
The only planes in TD are the vehicle production plane and a10 which are both uncontrollable support units.  The c17 definitely shouldn't count, and IMO the a10 shouldn't either.

Reply:
Dropped.